examples/stable-diffusion/training/train_text_to_image_sdxl.py

```diff
@@ -767,7 +776,20 @@ def main(args):
     )
 
     # Load scheduler and models
-    noise_scheduler = DDPMScheduler.from_pretrained(args.pretrained_model_name_or_path, subfolder="scheduler")
+    if args.scheduler == "euler_discrete":
+        noise_scheduler = GaudiEulerDiscreteScheduler.from_pretrained(args.pretrained_model_name_or_path, subfolder="scheduler")
+    elif args.scheduler == "euler_ancestral_discrete":
+        noise_scheduler = GaudiEulerAncestralDiscreteScheduler.from_pretrained(args.pretrained_model_name_or_path, subfolder="scheduler")
+    elif args.scheduler == "ddim":
+        noise_scheduler = GaudiDDIMScheduler.from_pretrained(args.pretrained_model_name_or_path, subfolder="scheduler")
+    else:
+        noise_scheduler = DDPMScheduler.from_pretrained(args.pretrained_model_name_or_path, subfolder="scheduler")
```

Comment:
Do we want non-Gaudi enabled `DDPMScheduler` as default?

Reply:
This was default in original code, should we remove it??

Reply:
Ok if it was in original code, keep it :)

Reply:
But then allow also override option from user so it can be forced, regardless of model card definition